### PR TITLE
Added sections on identifiers and proofs

### PR DESCRIPTION
**Pull request by deiu**
I've started to incorporate some of the changes we discussed during our last f2f, specifically:
* adding an intro section
* adding section that mentions the upcoming DID WG and its connection with the VCWG
* adding section that mentions proof syntax and trade-offs (table) 

Please label this PR as `pending`.

### CODE REVIEW

**Comment by burnburn** [comment]
Everyone please review!!

**Comment by TallTed** [comment]
I think @deiu should fix the conflicts (might be just a rebase? tho probably after #13 is merged) before the rest of us review...

**Comment by deiu** [comment]
I've fixed the (minor) merge conflict. Please go ahead and review the PR.

File: index.html [hunk 1]
```diff
@@ -411,17 +411,18 @@ <h2>Proofs</h2>
...
-          All object properties in a JSON-LD document, such as the property "homepage",
-          are either keywords or they are mapped to an IRI. This feature enables open
-          world systems to identify the semantic meaning of the property in an unambiguous
-          way, which enables seamless merging of data between disparate systems.
           JSON object properties are not mapped to IRIs, which result in ambiguities with
           respect to the semantic meaning of the property. For example, one JSON document
           can use "title" (meaning "book title") in a way that is semantically
           incompatible with another JSON document using "title" (meaning "job title").
+          In contrast, all object properties in a JSON-LD document, such as the property "homepage",
+          are either keywords or mapped to an IRI. This feature enables open
```
Comment (TallTed) [inline index.html]:
```pre
In contrast, all object properties in a JSON-LD document, such as the property "homepage",
are either keywords or mapped to an IRI.
```
-> 
```pre
In contrast, each and every object property in a JSON-LD document, such as the property 
"homepage", is either a keyword, an IRI, or a literal mapped to an IRI via a context.
```
(`each and every` could be just `each`)

**Review by TallTed — APPROVED** [review]

**Pull request merged**